### DISCUSSION TITLE
Thank you

### DISCUSSION THREAD

**Discussion by deniselemonaki** [EddieHubCommunity/support#122] [General]
Thank you and Ruth for the livestream today on how to contribute to open source as a beginner and all the work you do

**Comment by eddiejaoude** [comment]
Thank you @deniselemonaki for the positive feedback. You are right @Ruth-ikegah live stream was awesome 🔥 .

Any questions or anything we can do to help you on Open Source, let us know 🤓 . Our community is really helpful 💪

**Reply by eddiejaoude** [reply]
Invitation sent for the GitHub Organisation. Welcome to the community 🎉

Don't forget after accepting to make it public so it appears on your github profile for everyone else to see, you can do this by finding your name in the github organisation list and change the dropdown to public https://github.com/orgs/EddieJaoudeCommunity/people

Tips for practicing:

Customise your GitHub profile, here is a live stream on it https://www.youtube.com/watch?v=cT6GXCuS0Zo
Practice repo, instructions how to add your name to the README in the README https://github.com/EddieJaoudeCommunity/hacktoberfest-practice
Remember contributing to open source is not just about code, its about collaboration, communication and adding value

I hope that helps
